src/bp/ui-studio/src/web/components/Layout/StatusBar/index.jsx

```diff
@@ -146,6 +164,16 @@ class StatusBar extends React.Component {
           <ActionItem title="Notification" description="View Notifications" className={style.right}>
             <NotificationHub />
           </ActionItem>
+          <ActionItem
+            title="NLU Performance Status"
+            description={
+              this.state.nluStatus.f1score ? `f1: ${this.state.nluStatus.f1score}` : 'currently no f1 to display'
+            }
+            disabled={this.state.nluStatus.computing}
+            className={style.right}
+          >
+            <NluPerformanceStatus synced={this.state.nluStatus.synced} updateNluStatus={this.updateNluStatus} />
+          </ActionItem>
```

Comment:
I was wondering if it was possible to keep all the nlu status logic inside your NluPerformanceStatus component. I know you have some parameters used to display information, but you could do something like this:

```js
   // Replace the <ActionItem part by this:
  <NluPerformanceStatus />

  // Then in your tsx component:
  render(){
    return (<ActionItem> ... your actual component</ActionItem>)
}
```

The train event can be passed down to the component to invalidate the sync status